seeds/01-dev.ts

```diff
@@ -0,0 +1,34 @@
+import { Knex } from "knex";
+
+
+//adds users, API Keys and assigns 1 CREATE_API_KEY role to first API key
+
+export async function seed(knex: Knex): Promise<void> {
+    // Deletes existing entries for clean slate (testing environment)
+    await knex("nerfbot.roles").del();
+    await knex("nerfbot.api_keys").del();
+    await knex("nerfbot.users").del();
+
+
+    // Inserts users into users table
+    await knex("nerfbot.users").insert([
+        { userId: 1, username: "mario"},
+        { userId: 2, username: "luigi"},
+        { userId: 3, username: "yoshi"}
+    ]);
+
+
+    // Inserts API Keys into api_keys table
+    await knex("nerfbot.api_keys").insert([
```

Comment:
`apiKeyId` will be auto generated by postgres, so you only have to pass in `userId`